app/Http/Controllers/Import/ImportFailureController.php

```
@@ -0,0 +1,514 @@
+
+        if ($request->has('status') && $request->filled('status')) {
+            $query->where('status', $request->input('status'));
+        }
+
+        $filename = "import-{$import->id}-failures-".now()->format('Y-m-d-H-i-s').'.csv';
+
+        return response()->stream(function () use ($query) {
+            $file = fopen('php://output', 'w');
+
+            // Write CSV headers
+            fputcsv($file, [
```

PHPStan (2.1.15)

Parameter #1 $stream of function fputcsv expects resource, resource|false given.


(argument.type)